website/publish/multi_website.rst

```diff
@@ -37,25 +37,25 @@ Then, select a theme. This new website might have an entirely different
 purpose or audience than the first one. So feel free to go for a
 different theme!
 
-Once the theme is selected, you can start to build the homepage of your
+Once the theme is selected, you can start to build the homepage for your
 website. Follow the purple drops; they will help you in the first steps.
 
-.. image:: media/multi_website08.png
+.. image:: media/updated_website_builder.png
```

Comment:
Please use [pngquant](https://www.odoo.com/documentation/user/14.0/contributing/documentation/introduction_guide.html#contributing-pngquant) to compress this file! 1.4 MB is quite large for a simple image. Merging this would add weight to the repository itself and slow down the rendering of the documentation page.

Reply:
will do